### PR TITLE
Add Docker files

### PR DESCRIPTION
**Pull request by mpgxvii**
- Add Docker files and Docker config files for the TimescaleDB connector

### CODE REVIEW

File: docker/sink-timescale.properties.template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name = radar-timescale-sink
+connector.class = io.confluent.connect.jdbc.JdbcSinkConnector
+tasks.max = 2
+transforms = recordTransform
+transforms.recordTransform.type = org.radarbase.kafka.connect.transforms.KeyValueTransform
```
Comment (blootsvoets) [inline docker/sink-timescale.properties.template:5]: As a check: is the transform jar already added to the JDBC connector jar?

**Review by blootsvoets — APPROVED** [review]
LGTM! Nice work!

File: docker/sink-timescale.properties.template (hunk 1 above)
Comment (mpgxvii) [reply docker/sink-timescale.properties.template:5]: It's just added to the same `CONNECT_PLUGIN_PATH`. The Dockerfile is here: https://github.com/RADAR-base/RADAR-JDBC-Connector/blob/docker/Dockerfile. Is this okay?

File: docker-compose.yml [hunk 2]
```diff
@@ -22,6 +199,7 @@ services:
     image: grafana/grafana:7.1.0-beta1
     depends_on:
       - timescaledb
+      - radar-jdbc-connector
```
Comment (nivemaham) [inline docker-compose.yml:202]: shouldn't be other way around?
The connector depends on database.

File: docker-compose.yml [hunk 3]
```diff
@@ -6,6 +6,183 @@ volumes:
...
+      - ./docker/sink-timescale.properties:/etc/kafka-connect/sink-timescale.properties
+    depends_on:
+      - zookeeper-1
+      - zookeeper-2
+      - zookeeper-3
+      - kafka-1
+      - kafka-2
+      - kafka-3
+      - schema-registry-1
+      - timescaledb
```
Comment (mpgxvii) [inline docker-compose.yml:164]: > shouldn't be other way around?
> The connector depends on database.


@nivemaham Did you mean here?

**Pull request merged**